src/main/java/group1/intern/controller/admin/AccountsManagementController.java

```diff
@@ -7,7 +7,7 @@
 import group1.intern.annotation.PreAuthorizeAdmin;
 
 @Controller
-public class AccountsController {
+public class AccountsManagementController {
     @PreAuthorizeAdmin

```

Comment:
```suggestion
    @PreAuthorizeAdmin
```
cho lên trên Controller luôn